cpp/daal/src/algorithms/dtrees/dtrees_train_data_helper.i

```diff
@@ -147,9 +160,10 @@ public:
         DAAL_ASSERT(_indexedFeatures);
         return *_indexedFeatures;
     }
-    void init(const NumericTable * data, const NumericTable * resp)
+    void init(const NumericTable * data, const NumericTable * resp, const NumericTable * weight = nullptr)
```

Comment:
weights?

Reply:
it was fixed, here and in other similar places